chapters/en/slides/module5/module5_05.Rmd

```diff
@@ -0,0 +1,393 @@
+---
+params:
+  dynamictitle: "module5_05"
+title: "`r params$dynamictitle`"
+output:
+  md_document:
+    variant: gfm
+    pandoc_args: "--markdown-headings=atx"
+---
+
+```{r setup, include=FALSE}
+source('../../../../slide-setup.R')
+# Remember to also manually update the YAML title above
+```
+
+type: slides
+
+
+# Effective use of color for quantitative data
```

Comment:
Please use Canadian spelling of colour. I will leave the rest for you to change.

```suggestion
# Effective use of colour for quantitative data
```